pkg/kubectl/cmd/get.go

```diff
@@ -262,10 +279,13 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 
 	for ix := range objs {
 		var mapping *meta.RESTMapping
+		var original runtime.Object
```

Comment:
And the reason seems to be that all object printers in pkg/kubectl/resource_printer.c seem to be operating on internal types unless I am missing something.